login/pam.c

```diff
@@ -37,49 +37,43 @@ static int parse_pam_args(pam_handle_t *pamh, int argc, const char **argv,
                           glome_login_config_t *config) {
   memset(config, 0, sizeof(glome_login_config_t));
   int errors = 0;
+  status_t status;
 
   for (int i = 0; i < argc; ++i) {
     if (!strncmp(argv[i], "config_path=", 12)) {
```

Comment:
I'm not saying that it'd be better readable but the offsets make me all queasy. Would it make sense to replace the string constants with `const char kConfigPath[] = "config_path=";` like constants and use sizeof on the constant instead of the hardcoded numbers?

Reply:
Sounds good, I added a helper function to handle this.